### ISSUE TITLE
When `rails new`, one file is not commited

### ISSUE THREAD

**Issue by db0sch** [damienlethiec/modern-rails-template#16]
`.overcommit.yml` is not commited after the rails new process

**Comment by damienlethiec** [comment]
Yep, for the moment it is normal. We need to find a workaround. Creating `.overcommit.yml` setup git hook. One of then check rubocop errors before commit. So you cannot commit as long as you have rubocop errors. And their is still something like 15 rubocop errors in the project that you need to fix manually before being able to commit again.

Not sure I am clear.

The only workaround I see would be to ignore the rubocop errors generated by rails new in the template. They are mostly LineTooLong errors in config files.
